test/numbers.jl

```diff
@@ -830,6 +830,26 @@ end
     @test cmp(isless, 1, NaN) == -1
     @test cmp(isless, NaN, NaN) == 0
 end
+@testset "ispositive/isnegative" begin
+        for T in [Base.uniontypes(Base.BitInteger)..., Bool, Rational{Int}, BigInt, Base.uniontypes(Base.IEEEFloat)..., BigFloat, Missing]
+        values = T[zero(T), one(T)]
```

Comment:
Am I missing something or indentation here is off and the closing `end`s below are correctly indented just because above you used 8 spaces?  But probably fixing the indentation above would solve this issue too.